include/cadmium/engine/pdevs_dynamic_runner.hpp

```diff
@@ -61,6 +61,8 @@ namespace cadmium {
             class runner {
                 TIME _next; //next scheduled event
 
+                bool progressBar = false;
```

Comment:
How do you select the value of `progressBar`? Do you change it manually in this file?
I think we must think of a better way of setting this.

Reply:
They are set using 2 small functions at the very bottom. We could change it to be a param for the runUntil() function tough.